### ISSUE TITLE
Add `watchdog` test after kernel crash

### ISSUE THREAD

**Issue by Psotas** [Dasharo/open-source-firmware-validation#215]
### The problem you're addressing (if any)

No `watchdog option` tests

**Comment by macpijan** [comment]
Related ticket: https://github.com/Dasharo/dasharo-issues/issues/734

**Comment by PLangowski** [comment]
@macpijan could you provide a description of what should be done here?

**Comment by artur-rs** [comment]
@PLangowski issue linked by Maciej shows the entire test case with steps to introduce the new test. First, check and propose the best place for the new test, then implement and verify it.

**Comment by macpijan** [comment]
Sometimes it is a matter of checking what's there and some issues might be fixed already. For watchdog, it seems we already have test here: https://github.com/Dasharo/open-source-firmware-validation/blob/develop/dasharo-compatibility/watchdog.robot

**Comment by PLangowski** [comment]
@macpijan We don't have a test that verifies that the watchdog resets the machine after a kernel crash.

**Comment by PLangowski** [comment]
Unless it's the same case as in the second test

**Comment by macpijan** [comment]
Right, got misled by the issue name. Adding a test after kernel crash would make very much sense. Reopening.

**Comment by macpijan** [comment]
In terms of firmware, you should be able to get one from the CI in the linked PR.

**Comment by PLangowski** [comment]
Yes, I've been searching for the artifacts from the [deployment](https://github.com/Dasharo/coreboot/actions/runs/8689984855/job/23828824675), but I can't find any

**Comment by SebastianCzapla** [comment]
Closing, as the PR with solution was merged.
